dask_sql/physical/rel/logical/window.py

```diff
@@ -171,13 +161,13 @@ def map_on_each_group(
     if lower_bound.is_unbounded and (
         upper_bound.is_current_row or upper_bound.offset == 0
     ):
-        windowed_group = partitioned_group.expanding(min_periods=0)
+        windowed_group = partitioned_group.expanding(min_periods=1)
     elif lower_bound.is_preceding and (
         upper_bound.is_current_row or upper_bound.offset == 0
     ):
         windowed_group = partitioned_group.rolling(
             window=lower_bound.offset + 1,
-            min_periods=0,
+            min_periods=1,
```

Comment:
Just curious why this was changed to 1? Just looking to understand nothing "wrong" with that

Reply:
`min_periods` is the minimum number of observations a window needs to have for it to be valid else it's populated with nan/null. This matched the behavior we see with sql windows as well for some aggregations with empty windows. 

The existing logic written for calcite didn't need to do this since the calcite planner explicitly adds a count aggregation and clause checking for empty windows and adding nulls like this.
```
DaskProject(EXPR$0=[CASE(>($1, 0), $2, null:BIGINT)])
  DaskWindow(window#0=[window(aggs [COUNT($0), $SUM0($0)])])
```